src/components/CompareResults/Retrigger/RetriggerButton.tsx

```diff
@@ -84,6 +84,10 @@ function RetriggerButton({ result }: RetriggerButtonProps) {
     enqueueSnackbar(message, {
       variant: 'info',
       autoHideDuration: 10000, // The default (6000ms) configured in App.tsx seems a bit low for this notification
+      // It's common that 2 notifications are sent (one for base, one for
+      // new), so specifying the width makes them look more consistent.
+      // This maxWidth is copied from notistack's own style.
+      style: { width: '1000px', maxWidth: 'calc(100vw - 40px)' },
```

Comment:
@Carla-Moz I came up with this to fix the width of the notifications while still making them OK in small screens. Please tell me if that's OK for you! I couldn't find anything better, especially "100%" for maxWidth wasn't working...

Reply:
Tested on my local and it looks great. Thank you!